src/concrete/ml/sklearn/base.py

```diff
@@ -580,6 +638,22 @@ def compile(
         # Enable evaluation key compression
         enable_key_compression = os.environ.get("USE_KEY_COMPRESSION", "1") == "1"
 
+        if ciphertext_format == CiphertextFormat.TFHE_RS:
+            assert all(
+                (
+                    self.input_quantizers[i].is_signed == self.input_quantizers[0].is_signed
+                    for i in range(1, len(self.input_quantizers))
+                )
+            )
+            is_signed = self.input_quantizers[0].is_signed
+
+            crypto_params = json.loads(fhext.get_crypto_params_radix())  # pylint: disable=no-member
```

Comment:
why do we use deai crypto params ?

Reply:
we need to get the radix parameters from somewhere. Concrete doesn't set the parameters, so Concrete ML must set them, using default parameters from Rust. Fhext is the only Rust package where we can set these parameters.

Reply:
Got it